cmake/FindHIPCUB.cmake

```diff
@@ -22,8 +22,10 @@ endif()
 # For CMake >= 3.12, this will also check CUB_ROOT and ENV{CUB_ROOT}
 find_path(HIPCUB_INCLUDE_DIR
           NAMES hipcub/hipcub.hpp
-          PATHS ${HIP_ROOT_DIR} /opt/rocm/include
-                ENV ROCM_PATH)
+          PATHS ${HIP_ROOT_DIR}
+                /opt/rocm
```

Comment:
I assume CMake searches these in the order given. Does it make sense to check `/opt/rocm` before `ROCM_PATH`? Should we flip these?

Reply:
Yes.